USAGE_desktop_D3D12.md

```diff
@@ -285,16 +285,15 @@ Vulkan-only:
                                         and suballocations.
 
 D3D12-only:
-  --dcp                 Force CachedPSO to null when creating graphics or compute PSOs.
-                        Can help enable replay across changing driver installs.
-                        (Same as --discard-cached-psos)
-  --debug-device-lost   Enables automatic injection of breadcrumbs into command buffers
-                        and page fault reporting.
-                        Used to debug Direct3D 12 device removed problems.
-  --fw <width,height>   Setup windowed and override resolution.
-                        (Same as --force-windowed)
-  --create-dummy-allocations Enables creation of dummy heaps and resources
-                             for replay validation.
+  --use-cached-psos            Permit using cached PSOs when creating graphics or compute pipelines.
+                               Disabling helps enable replay across changing driver installs.
```

Comment:
Can this text be changed to give some more detail about why a user might or might not want to enable the flag, for example:
```
  --use-cached-psos   Permit using cached PSOs when creating graphics or compute pipelines.
                      Using cached PSOs may reduce PSO creation time but may result in replay errors.
```

`PrintUsage()` in `replay_settings.h` also needs to be updated to match whatever text is used here.